frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -375,27 +371,27 @@ export default class ArenaContestList extends Vue {
         sortBy = (a, b) =>
           compareNumber(a.participating ? 1 : 0, b.participating ? 1 : 0);
         break;
+      default:
+        sortBy = (a, b) => a.title.localeCompare(b.title);
+        break;
     }
-    this.contestList.sort(sortBy);
+    return this.contestList.sort(sortBy);
```

Comment:
```suggestion
    return this.contestList.slice().sort(sortBy);
```

esto hace que `this.contestList` no se toque para nada.